jdaviz/configs/cubeviz/plugins/sonify_data/sonify_data.py

```diff
@@ -78,12 +78,22 @@ def __init__(self, *args, **kwargs):
 
         self.results_label_default = 'Sonified data'
         self.add_to_viewer_selected = 'flux-viewer'
+        self._results_label_counter = 1
 
     @property
     def user_api(self):
         expose = ['sonify_cube']
         return PluginUserApi(self, expose)
 
+    @observe('results_label_invalid_msg')
+    def _avoid_temporary_vue_error(self, event):
```

Comment:
could this be triggered by `DataCollectionAddMessage` instead and count from the `data_collection` itself instead of having an internal increment?  Although it isn't a common use-case, plugins can be instantiated _after_ the app is initialized, which would break the counter here (imagine doing one sonification and then instantiating a new instance of the plugin, that default label would then be incremented incorrectly).